test/integration/util/integration_logger.go

```diff
@@ -0,0 +1,103 @@
+// +build integration
+
+/*
+Copyright (C) 2017 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package util
+
+import (
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"path"
+	"strings"
+)
+
+const (
+	messageInfoMaxLength = 7
+	timeHeaderLength     = 16
+)
+
+var (
+	IntegrationLog *log.Logger
+	logFile        *os.File
+)
+
+func init() {
+	IntegrationLog = log.New(ioutil.Discard, "", 0)
```

Comment:
The reason is that before the beforeScenario(), where testDir is set, there is a call for isCDK() in testMain(). And there it would fai without init(), because isCDK() runs `minishift -h` where it searches for setup-cdk command and it would log into non-existing Logger.

Second option I had in my head was to add a check into LogMessage(), but this would need manual duplicate setup of value which would be equal to testDir which is not good for maintance and also it would not prevent the use of util.ntegrationLog.Print(). So I went for init(). (However I am not sure if there are any costs for doing init(). Seemed fine to me.)